contracts/uniswap_exchange.vy

```diff
@@ -135,8 +140,8 @@ def getInputPrice(input_amount: uint256, input_reserve: uint256, output_reserve:
 @constant
 def getOutputPrice(output_amount: uint256, input_reserve: uint256, output_reserve: uint256) -> uint256:
     assert input_reserve > 0 and output_reserve > 0
-    numerator: uint256 = input_reserve * output_amount * 1000
-    denominator: uint256 = (output_reserve - output_amount) * 997
+    numerator: uint256 = input_reserve * output_amount * self.swap_fee
+    denominator: uint256 = (output_reserve - output_amount) * 1000
```

Comment:
Shouldn't this be reversed?

Reply:
yes becuese it will be confusing for the user for platform fee enter number between 0 - 1000
and for swap fee enter just percent.
i think it better to keep the same idea for both.